web/geolocation/geolocation.css

```diff
@@ -0,0 +1,227 @@
+body {
+    font-family: 'Poppins';
```

Comment:
Eventually, we should have a consistent body style, defined in shared, but let's see what Monika comes back with for this first.